chocolatey/plugins/modules/win_chocolatey.py

```diff
@@ -99,7 +105,7 @@
     - Should install arguments be used exclusively without appending
       to current package passed arguments.
     type: bool
-    version_added: '2.10'
+    version_added: '1.1.0'
```

Comment:
Looks like you changed the wrong version here? The one you initially added was on line 88 🙂